testing/sdk_utils.py

```diff
@@ -56,13 +59,40 @@ def dcos_version_less_than(version):
     return shakedown.dcos_version_less_than(version)
 
 
+def set_test_index(index):
+    '''Assigns the index to use for a test within a given test suite.
+    Should start at 1 for the first test in the suite.'''
+    global test_index
+    test_index = index
+
+
+def get_test_suite_name(pytest_node):
+    '''Returns the test suite name to use for a given test.'''
+    # frameworks/template/tests/test_sanity.py => test_sanity_py
+    # tests/test_sanity.py => test_sanity_py
+    return pytest_node.parent.name.split('/')[-1].replace('.','_')
+
+
+def get_test_suite_log_directory(pytest_node):
+    '''Returns the parent directory for the logs across a suite of tests.
+    For individual tests within this directory, see get_test_log_directory().'''
+    return os.path.join('logs', get_test_suite_name(pytest_node))
+
+
 def get_test_log_directory(pytest_node):
+    '''Returns the directory for the logs of a single test.
+    For the parent test suite directory, see get_test_suite_log_directory().'''
     # full item.listchain() is e.g.:
     # - ['build', 'frameworks/template/tests/test_sanity.py', 'test_install']
     # - ['build', 'tests/test_sanity.py', 'test_install']
     # we want to turn both cases into: 'logs/test_sanity_py/test_install'
-    parent_name = pytest_node.parent.name.split('/')[-1].replace('.','_')
-    return os.path.join('logs', parent_name, pytest_node.name)
+    if test_index >= 0:
```

Comment:
Why do we check for 0 if we have: "Should start at 1 for the first test in the suite" where this is referenced above? Also, should we not explicitly indicate that this is a global?

Reply:
`global` not required if the value is only being read

Reply:
...though it would be nice to be explicit here on where `test_index` is coming from.

Reply:
Done. Added `global` statement, checking `> 0`